### ISSUE TITLE
Multilingual assets

### ISSUE THREAD

**Issue by PiKeyAr** [PiKeyAr/sadx-hd-gui#5]
**Adventure tutorial texts**
- [x] Japanese
- [x] English
- [ ] French
- [x] Spanish
- [ ] German

**Rank A/B/C mission cards**
- [x] Japanese
- [x] English
- [ ] French
- [ ] Spanish
- [ ] German

**File select dialog `AVA_FSDLG` PVMs**
- [x] Japanese
- [x] English
- [ ] French
- [ ] Spanish
- [ ] German

**`CHAO_HYOUJI` PVMs (most likely unused in vanilla SADX)**
- [ ] Japanese
- [ ] English
- [ ] French
- [ ] Spanish
- [ ] German

> It's also possible to add more translated assets like SADX Steam does:
> - "TUTO_CMN" PVMs
> - "AVA_DLG" PVMs
> - "AVA_FILESEL" PVMs
> - "AVA_FSDLG" PVMs
> - "AVA_GTITLE0" PVMs
> - "AVA_TITLE" PVMs
> - "AVA_VMSSEL" PVMs
> - "CON_REGULAR" PVMs
> - "GAMEOVER" PVMs

**Comment by PiKeyAr** [comment]
Turns out most of that additional stuff doesn't even work in the Steam version, and several of the "translated" PVMs are either incomplete or simply duplicates of the English version. The only thing that is fully functional in the Steam version is the CON_REGULAR translation it seems.

**Comment by PiKeyAr** [comment]
**How to make French or German assets for HD character tutorials**

To create HD character tutorials, start by making texture packs out of the following PVMs in the mod's `system` folder:

French:
```TUTOMSG_AMY_F_HD.PVM
TUTOMSG_BIG_F_HD.PVM
TUTOMSG_E102_F_HD.PVM
TUTOMSG_KNUCKLES_F_HD.PVM
TUTOMSG_SONIC_F_HD.PVM
TUTOMSG_TAILS_F_HD.PVM
```

German:
```TUTOMSG_TAILS_G_HD.PVM
TUTOMSG_SONIC_G_HD.PVM
TUTOMSG_AMY_G_HD.PVM
TUTOMSG_BIG_G_HD.PVM
TUTOMSG_E102_G_HD.PVM
TUTOMSG_KNUCKLES_G_HD.PVM
```

Using the HD English, Japanese or Spanish version (located in the mod's `textures` folder) as an example, upscale and/or recreate the assets in 2x resolution. Use the assets below to make it consistent with other translations.

Source assets for translators (unsorted)
https://dcmods.unreliable.network/owncloud/data/PiKeyAr/files/Various/HDGUIStuff.zip
https://dcmods.unreliable.network/owncloud/data/PiKeyAr/files/Various/HDGUI_tuto.7z

**Comment by PiKeyAr** [comment]
Closed in favor of #8 and #7
